docs/source/el7toel8/envars.md

```diff
@@ -8,11 +8,18 @@ for the leapp project to distinguish their purpose: *production* or *devel* use.
 If the argument for envars below is not specified, it is expected to set `0`
 (false) or `1` (true).
 
+
+## LEAPP_GRUB_DEVICE
+
+Overrides the automatically detected GRUB core location.
```

Comment:
```suggestion
Overrides the automatically detected storage device with GRUB core (e.g. /dev/sda).
```

Reply:
Ah, my bad. I thought it referred to the GRUB core directory itself.